deployer/README.md

```diff
@@ -1175,3 +1175,260 @@ the first created. You can check if the route in question is defined in multiple
+                  value: <logging-es | logging-es-ops>
+                - name: "OPS_PORT"
+                  value: 9200
+                - name: "OPS_CLIENT_CERT"
+                  value: /etc/fluent/keys/cert
+                - name: "OPS_CLIENT_KEY"
+                  value: /etc/fluent/keys/key
+                - name: "OPS_CA"
+                  value: /etc/fluent/keys/ca
+              volumes:
+              - name: runlogjournal
+                hostPath:
+                  path: /run/log/journal
+              - name: varlog
+                hostPath:
+                  path: /var/log
+              - name: varlibdockercontainers
+                hostPath:
+                  path: /var/lib/docker/containers
+              - name: config
+                configMap:
+                  name: logging-fluentd-enrichment
+              - name: certs
+                secret:
+                  secretName: logging-fluentd
+              - name: dockerhostname
+                hostPath:
+                  path: /etc/hostname
+              - name: localtime
+                hostPath:
+                  path: /etc/localtime
+              - name: dockercfg
+                hostPath:
+                  path: /etc/sysconfig/docker
+
+Once `enrichment-fluentd-dc.yaml` has been created, run the following to create
+the new DC:
+
+    $ oc create -f enrichment-fluentd-dc.yaml
+
+Now that the enrichment Fluentd has been set up, the original Fluentd will need to
+be updated.
+
+Edit the provided configmap for Fluentd:
+
+    $ oc edit configmap/logging-fluentd
+
+Update it so the fluent.conf section looks like the following:
+
+    # This file is the fluentd configuration entrypoint. Edit with care.
+
+    @include configs.d/openshift/system.conf
+
+    # In each section below, pre- and post- includes don't include anything initially;
+    # they exist to enable future additions to openshift conf as needed.
+
+    ## sources
+    ## ordered so that syslog always runs last...
+      @include configs.d/openshift/input-pre-*.conf
+      @include configs.d/dynamic/input-docker-*.conf
+      @include configs.d/dynamic/input-syslog-*.conf
+      @include configs.d/openshift/input-post-*.conf
+    ##
+
+    <label @INGRESS>
+      <match **>
+        @type secure_forward
+
+        self_hostname ${HOSTNAME}
+        shared_key <our generated string>
+
+        secure yes
+        ca_cert_path        /etc/fluentd/keys/ca
+
+        <server>
+          host logging-enrichment-fluentd.logging.svc.cluster.local
+          port 24284
+        </server>
+      </match>
+    </label>
+
+Next, since our collecting Fluentd expects to ship its logs to a service called
+'logging-enrichment-fluentd' that service will need to be created:
+
+    $ oc expose dc logging-enrichment-fluentd --port=24284
+
+At this point, the enrichment Fluentd pods can be scaled up. They do not need to
+be 1:1 with the Fluentd collectors, A ratio of 1:10 is likely a good start:
```

Comment:
I don't think it would work.  We should not recommend using in_multiprocess unless we can support it, either we provide it or allow the customer to install it, and actually try it out in house and verify exactly what the performance improvement is.  In the testing by @lukas-vlcek he could not figure out how to use it to gain an appreciable performance gain in the openshift aggregated logging case.